common/supplemental/likelySubtags.xml

```diff
@@ -9122,9 +9109,9 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<likelySubtag from="jib" to="jib_Latn_NG" origin="sil1"/>	<!-- Jibu ➡︎ Jibu (Latin, Nigeria) -->
 		<likelySubtag from="jra" to="jra_Latn_VN" origin="sil1"/>	<!-- Jarai ➡︎ Jarai (Latin, Vietnam) -->
 		<likelySubtag from="jra_Khmr" to="jra_Khmr_KH" origin="sil1"/>	<!-- Jarai (Khmer) ➡︎ Jarai (Khmer, Cambodia) -->
-		<!-- CLDR-16405 the following two conflict with the metadata aliases treating aju as the primary child of jrb -->
-		<!-- <likelySubtag from="jrb" to="jrb_Hebr_IL" origin="sil1"/> -->	<!-- Judeo-Arabic ➡︎ Judeo-Arabic (Hebrew, Israel) -->
-		<!-- <likelySubtag from="jrb_Arab_MA" to="jrb_Arab_MA" origin="sil1"/>-->	<!-- Judeo-Arabic (Arabic, Morocco) ➡︎ Judeo-Arabic (Arabic, Morocco) -->
+		<likelySubtag from="jrb" to="jrb_Hebr_IL" origin="sil1"/> 	<!-- Judeo-Arabic ➡︎ Judeo-Arabic (Hebrew, Israel) -->
```

Comment:
This one I am unsure about. We have `aju` as the primary child of `jrb` and the default script for `aju` is I think supposed to be `Arab`. But we will see how this goes during the next ICU integration, at worst we might have to comment out this line again.

Reply:
Hmmm. We've introduced direct inheritance from language A to language B (eg nb -> no), but I don't think we have a test that the maximized script doesn't change in doing so, which is pretty fragile. Should probably add such a test, which would force us to change, eg from aju -> jrb into aju -> jrb_Arab.

Note, according to Wikipedia, it appears that https://en.wikipedia.org/wiki/Judeo-Arabic_dialects were consistently written in Hebrew